src/index.tsx

```diff
@@ -8,9 +8,7 @@ const root = ReactDOM.createRoot(
     document.getElementById("root") as HTMLElement,
 );
 root.render(
-    <React.StrictMode>
```

Comment:
Remind me why you are making this change?

Also, don't forget to `npm run prettier`

Reply:
Will run when I get back home.